mne/time_frequency/tests/test_tfr.py

```diff
@@ -264,7 +276,12 @@ def test_tfr_multitaper():
                                             time_bandwidth=4.0, picks=picks)
     power_evoked = tfr_multitaper(epochs.average(), freqs=freqs,
                                   n_cycles=freqs / 2., time_bandwidth=4.0,
-                                  return_itc=False)
+                                  return_itc=False, average=False).average()
```

Comment:
This is a bit confusing to me. You're averaging over trials first, then calling the `tfr` function, then averaging again? What are you averaging over the second time, a single trial? Why not have a check somewhere for `if isinstance(inst, Evoked)` and make the output an AverageTFR regardless of whether `average=True`